java-api-core/src/com/kinvey/java/model/KinveyBatchInsertError.java

```diff
@@ -27,11 +27,11 @@ public void setCode(long code) {
         this.code = code;
     }
 
-    public long getIndex() {
+    public int getIndex() {
```

Comment:
since this is a `public` member, this will be a breaking change right? Is there any way to avoid this breaking change?

Reply:
fixed